### ISSUE TITLE
Enable convert Set to array `unicorn/prefer-spread`

### ISSUE THREAD

**Issue by tiagoporto** [sindresorhus/eslint-plugin-unicorn#1032]
`Array.from` is useful to convert a `new Set()`.

Approach could be used to remove duplicate items from array and continue using array methods.

```js
const fruits = Array.from(new Set(['banana', 'apple', 'banana']))

fruits.map(fruit => console.log(fruit))
```

Also it's good disabled as fixable, after fixed the code the change broke my application.

**Comment by fisker** [comment]
What do you mean?

```js
const fruits = Array.from(new Set(['banana', 'apple', 'banana']))
```

is fixed to

```js
const fruits = [...new Set(['banana', 'apple', 'banana'])]
```

And they are have the same result.

**Comment by tiagoporto** [comment]
Hi @fisker 

Yes, the code was fixed to 

```js
const fruits = [...new Set(['banana', 'apple', 'banana'])]
```

But, while
```js
const fruits = Array.from(new Set(['banana', 'apple', 'banana']))
```
returns `['banana', 'apple']`

```js
const fruits = [...new Set(['banana', 'apple', 'banana'])]
```
I don't know why, returns ` [ Set(2) { 'banana', 'apple' } ]`

And `fruits.map` stopped work.

**Comment by sindresorhus** [comment]
> I don't know why, returns [ Set(2) { 'banana', 'apple' } ]

In what environment? Do you use a transpiler or bundler? This sounds like incorrect code transpilation.

**Comment by tiagoporto** [comment]
@sindresorhus you are right, babel is transpiling the code to `[].concat(new Set(["banana","apple"]))`.

Thank you
